### ISSUE TITLE
A11y_SQLAzureDataStudio_SchemaCompare_Comparison_ScreenReader: Unnecessary extra information announced by screen reader related to the whole screen.

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/azuredatastudio#9400]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11yMAS;#A11yTCS;#SQL Azure Data Studio;#DesktopApp;#Win32;#MAS1.3.1;#Benchmark; #A11y_SQLAzureDataStudioBenchmark;#A11ySev2;#NVDA;#JAWS;

**Environment Details:**
Version: 1.16.0-insider
Commit: 506c6a5
Date: 2020-02-17T05:43:18.399Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS reference: MAS1.3.1
Screen Reader: NVDA(2019.3.1),JAWS

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Schema Compare extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Schema compare from Context menu.
8. Navigate till Select Source or Select Target & hit enter.
9. Provide all the details for Source & Target.
10. Navigate till Ok button & hit enter.
11. Navigate till Compare menu from Schema compare Menu bar & hit enter.
12. Check if any unnecessary announcements are made by screen reader.

**Actual:**
Unnecessary data related to the page is announced by screen reader every time we check/un-check   the check box present on the screen.

**Issue observed with NVDA and JAWS.**

**Expected:**
Unnecessary data should not be read by the screen reader apart from the information that has changed on the page.

**User Impact:**
Screen reader users will get confused as the unnecessary data related to the page is announced with the check box's checked/unchecked state.

**Attachment link reference:**
[9400_A11y_SQLAzureDataStudio_Extension_SchemaCompare_Comparison_ScreenReader.zip](https://github.com/microsoft/azuredatastudio/files/4291377/9400_A11y_SQLAzureDataStudio_Extension_SchemaCompare_Comparison_ScreenReader.zip)

**Comment by alanrenmsft** [comment]
looks like nvda issue?

**Comment by alanrenmsft** [comment]
please check other screen readers to see whether it is also announcing the whole control hierarchy after the check box is checked. if not, please close this as external nvda issue

**Comment by Benjin** [comment]
Confirmed that this repros in Narrator as well.

**Comment by Benjin** [comment]
Using a mutation observer script (https://gist.github.com/smhigley/75fa44df8218d62714ea6df5bab353b5), I determined that these checkboxes are not getting recreated each state-change.  Makes sense, as DOM changes aren't getting reflected in dev tools until they're closed and reopened so how could it pick it up... but I'm not sure why _that's_ happening.

**Comment by Benjin** [comment]
After attempting to demo for our accessibility expert, it still failed in Narrator but was now working in NVDA.  Possible that an update resolved a bug.  Expert said that Narrator often has issues with Chromium apps in this area.  Please reopen if you're still seeing this issue in NVDA, as I no longer am.

**Comment by Harika1702** [comment]
<img width="960" alt="9400_A11y_Issue Fixed" src="https://user-images.githubusercontent.com/100270074/218073901-9dba522d-1e7c-4925-aa02-6af84050ff52.png">
Verified this issue in the below environment.
App name: Azure data studio.
Version: 1.41.1 (user setup)
Commit: 33dce95b152227c4f10f6251ab30690fe9895c0c
Date: 2023-01-28T00:46:39.274Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22623
GitHub Tags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
